mapping/requirements_officejs.json

```diff
@@ -980,7 +980,18 @@
 								}
 							}],
 							"availability": "GA"
-						}                        
+						},
+						{
+							"name": "SharedRuntime",
+							"apiVersion": "1.1",
+							"supportedProductVersions": [{
+								"from": {
+									"build": "16.35",
```

Comment:
We need to use Telemetry Version number such as 16.37.0.0 (default last 2 to zero). Please check MRO site to figure out the exact ver# for MacOS and iOS.

Reply:
Ok folks, I have to bed for the night soon. I will keep checking this for another hour or so. If it's after 10:30PM PST, I won't be on to make this change for you in the docs. What you'll need to do then is make a pr to the office-js-docs-pr repo with the new build number ([listed on the Shared Runtime requirement set page](https://docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/shared-runtime-requirement-sets)). You will need to push your branch into the master branch and then master into live. If you don't have the ability to do that for some reason, let me know in the morning before the 8 AM PST call so I can push it out. (Linda Lu or any member of the docs team should also be able to push to live as well, FYI).

Reply:
Update using MAC telemetry version 16.34.209.0